app/src/main/res/layout/fragment_add_new_login.xml

```diff
@@ -0,0 +1,253 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
This is pretty much identical to the "edit" fragment, right?

Reply:
Yes, it is. Except for hostname text is clickable and focusable.